docs/index.md

```diff
@@ -15,46 +15,48 @@ In order to integrate with Lead Safe, the health provider will need to make a RE
+| address1        | Text     | Address number, street direction, street name, street type | Patientprofile.address1                                                                                           |
+| address2        | Text     | Additional address information, e.g. apartment number      | patientprofile.address2                                                                                           |
+| city            | Text     |                                                            | patientprofile.city                                                                                               |
+| state           | Text     |                                                            | patientprofile.state                                                                                              |
+| zip             | Text     | 5 digit zip code                                           | patientprofile.Zip                                                                                                |
+| date_of_birth   | Date     | yyyy-mm-dd                                                 |                                                                                                                   |
+| gender          | Text     | M/F/U                                                      |                                                                                                                   |
+| race            | Text     | Code Values from Centricity                                | Standard ONC Race definitions <br> <br> **Code / description** <br> 1002-5 / American Indian or Alaska Native <br> 2028-9 / Asian <br> 2054-5 / Black or African American <br> 2076-8 / Native Hawaiian or Other Pacific Islander <br> 2106-3 / White <br> UNK / Unknown |
+| ethnicity       | Text     | Code Values from Centricity                                | Standard ONC Ethnicity definitions <br> <br> **Code / description** <br> 2135-2 / Hispanic or Latino <br> 2186-5 / Non Hispanic or Latino <br> UNK / Unknown |
+| Visit Array     |          |                                                            |                                                                                                                   |
+| visit.visit_id  | Numeric  | 16 Digit GE ID                                             | Unique GE ID for specific visit  (DocumentID)                                                                     |
+| visit.date      | Date     | yyyy-mm-dd hh:mm:ss.sss-hh:mm                              | RFC 3339 (ISO 8601 derivative).                                                                                   |
+| visit.location  | Text     |                                                            | Text Abbreviation for facility of visit location                                                                  |
```

Comment:
This says that it should be the text abbreviation for the visit, but the example shows a fully specified address. Which is it?